Quiz Master/src/screens/statistics/FinaleScreen.tsx

```diff
@@ -0,0 +1,26 @@
+import { useDispatch, useSelector } from "react-redux";
+import ChangePageButton from "../../components/ChangePageButton";
+import { Routes } from "../../constants/routes";
+import { resetStatistics } from "../../store/slices/statisticsSlice";
+import { RootState } from "../../store";
+
+const FinaleScreen = () => {
+  const stats = useSelector((state: RootState) => state.statistics);
+  const dispatch = useDispatch();
+
+  const handleReset = () => {
+    dispatch(resetStatistics()); // Resets the persisted statistics
+  };
+
+  console.log("stats:", stats)
```

Comment:
Don't leave the logs 😄

Reply:
This was the only one I left since we haven't done the statistics screen. The main browser I use also doesn't support the redux took kit which is kind of annoying.